test/sanitize-field-names/express.js

```diff
@@ -0,0 +1,92 @@
+'use strict'
+const { createAgentConfig } = require('./_shared')
+const agent = require('../..').start(createAgentConfig())
+const {
+  resetAgent,
+  assertRequestHeadersWithFixture,
+  assertResponseHeadersWithFixture,
+  assertFormsWithFixture
+} = require('./_shared')
+
+const test = require('tape')
+const request = require('request')
+const express = require('express')
+const bodyParser = require('body-parser')
+const fixtures = require('./_fixtures')
+
+function runTest (
+  t, expected, agentConfig, requestHeaders, responseHeaders, formFields, middleware = false
+) {
+  agent._config(agentConfig)
+  const app = express()
+  if (middleware) {
+    app.use(middleware)
+  }
+
+  // resets agent values for tests.  Callback fires
+  // after mockClient receives data
+  resetAgent(agent, (data) => {
+    const transaction = data.transactions.pop()
+    assertRequestHeadersWithFixture(transaction, expected, t)
+    assertResponseHeadersWithFixture(transaction, expected, t)
+    assertFormsWithFixture(transaction, expected, t)
+  })
+
+  // register request handler
+  app.post('/test', (req, res) => {
+    t.ok('received request', 'received request')
+    res.header(responseHeaders)
+    res.send('Hello World')
+  })
+
+  const server = app.listen(0, '0.0.0.0', () => {
+    const url = `http://${server.address().address}:${server.address().port}/test`
+    request.post(
+      url,
+      {
+        form: formFields,
+        headers: requestHeaders
+      },
+      function (error, response, body) {
+        if (error) {
+          t.error(error)
+        }
```

Comment:
nit:
```suggestion
        t.error(error)
```

With `t.error` you can drop the `if (error) {` guard.